data/data/aws/master/main.tf

```diff
@@ -89,7 +91,16 @@ resource "aws_network_interface" "master" {
     var.tags,
   )
 }
-  
+
+# NOTE(russellb) For some reason, I was not able to access get IPv6 addresses
+# on the resource, but was able to get them using the network interface data
+# source.
```

Comment:
did we open an issue with upstream?

Reply:
No, I had moved on once I had an easy enough workaround.  I can come back around to filing an upstream issue with a reproducer later, though.